tock/tock/signals.py

```diff
@@ -12,7 +12,7 @@ def successful_login(sender, request, user, **kwargs):
 
 
 def successful_logout(sender, request, user, **kwargs):
-    logger.info(f'Successful logout event for {user.username}.')
+    logger.info(f'Successful logout event for {user}.')
```

Comment:
Wouldn't it be better to leave this as `user.username`, or are you going for the `repr`/`str` defined on the model instead by doing this?

Reply:
The value that got sent over after the redirect was no longer a Django user but rather the string of the user. This was occurring after the redirect I believe. when I user `user.username` it was hitting an error. I figured it's also easier to just rely on the `str` defined on the model anyway here. It never felt right to drill down into when the string representation of the `User` model could change and be more useful.